public/static/docs/command-reference/remote/index.md

```diff
@@ -101,7 +101,7 @@ remote = myremote
 > [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).
 
 ```dvc
-$ dvc remote add mynewremote s3://mybucket/myproject
+$ dvc remote add -d mynewremote s3://mybucket/myproject
```

Comment:
let's change title above and mention explicitly at least that we are adding a default remote?

Reply:
Good point, haven't noticed that. Adjusted. Thanks!